common/messages/eob.go

```diff
@@ -20,7 +20,7 @@ import (
 // eLogger is for EOM Messages and extends packageLogger
 var eLogger = packageLogger.WithFields(log.Fields{"message": "EOM"})
 
-type EOM struct {
+type EOB struct {
```

Comment:
I assume EOB  means End of Block?

Reply:
Yes (But I didn't change all the comments, or even much of what factomd would insert into a EOM... We can do that later)